Jenkinsfile

```diff
@@ -257,7 +257,7 @@ ansiColor('xterm') {
 
           stage('install') {
             image.inside(DOCKER_RUN_OPTS) {
-              sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' > ${HOME}/.npmrc'
+              sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' >> ${HOME}/.npmrc'
```

Comment:
This should be a single `>`, otherwise, if the file continues to exist, it'll have multiple identical lines